catalyst-uploader.go

```diff
@@ -27,27 +30,60 @@ func run() int {
 		glog.Error(err)
 		return 1
 	}
+	vFlag := flag.Lookup("v")
+	fs := flag.NewFlagSet("catalyst-uploader", flag.ExitOnError)
+
 	// cmd line args
-	describe := flag.Bool("j", false, "Describe supported storage services in JSON format and exit")
-	timeout := flag.Duration("t", 30*time.Second, "Upload timeout")
-	flag.Parse()
+	version := fs.Bool("version", false, "print application version")
+	describe := fs.Bool("j", false, "Describe supported storage services in JSON format and exit")
+	verbosity := fs.String("v", "", "Log verbosity.  {4|5|6}")
+	timeout := fs.Duration("t", 30*time.Second, "Upload timeout")
+	storageFallbackURLs := CommaMapFlag(fs, "storage-fallback-urls", `Comma-separated map of primary to backup storage URLs. If a file fails uploading to one of the primary storages (detected by prefix), it will fallback to the corresponding backup URL after having the prefix replaced`)
```

Comment:
Will it be possible/easy to change the command line options? I thought we said that would require a mist change and therefore would go with an environment variable?